pkg/broker/filter/filter_handler.go

```diff
@@ -182,8 +182,8 @@ func (r *Handler) serveHTTP(ctx context.Context, event cloudevents.Event, resp *
 		// framework returns a 500 to the caller, so the Channel would send this repeatedly.
 		return nil
 	}
-	delete(originalV3.Extensions, ttlKey)
-	event.Context = originalV3
+	delete(originalV1.Extensions, ttlKey)
+	event.Context = originalV1
```

Comment:
Is there are reason why we bump all events to V1? Can't broker/trigger be just pass through?